docs/schema/reference.md

```diff
@@ -118,6 +118,14 @@ In OCDS 1.0, the details (address, contact point, etc.) of the organizations inv
 Note that the organization references allow, but deprecate, the fields for organization details.
 ```
 
+#### Example
+
+```{jsoninclude} ../examples/release_schema_reference/release_package.json
+:jsonpointer:
```

Comment:
I think we want the `jsonpointer` to be the field, to avoid the clutter from the package.

In this case, I think we can expand all the fields within the party.

It might be nice to just show one party as well (or to collapse the second party).

https://standard.open-contracting.org/staging/1348-schema-examples/en/schema/reference/#parties

Reply:
> In this case, I think we can expand all the fields within the party.

Makes sense; what about the fields for planning, tender, etc.? Do we want to expand them all as well? Or maybe all of them but the building blocks? e.g., planning/documents, milestones, etc

Reply:
And, what about the order of the fields? Can we preserve the same order as the schema definition, or do we sort the fields alphabetically (as per `ocdskit indent`)?

Reply:
And do we want the examples to have all the fields from the object? E.g., for tender, should the example contain all the existing fields from the schema?

Reply:
> what about the fields for planning, tender, etc.? Do we want to expand them all as well? Or maybe all of them but the building blocks?

I agree that we can collapse the multi-use sub-schema (periods, documents, milestones, etc.) to avoid repetition.

> what about the order of the fields?

`ocdskit indent` re-orders fields? It shouldn't. Anyway, I think it is best for the example file to use fields in schema order, for consistency.

> do we want the examples to have all the fields from the object?

We can have as many as makes sense. I think only the `tender` object has a large number of fields.

Reply:
Done